src/classes/application.js

```diff
@@ -160,6 +161,21 @@ module.exports = class Application extends Base {
     const app = express()
     _app.set(this, app)
 
+    log.trace('Ash server adding body parsing middleware')
+    const bodyParserOptions = config.bodyParser || {}
+    if (bodyParserOptions.json !== false) {
```

Comment:
My intention was that if undefined then the block would run. Which constitutes the default. 
Agree that some good checking and app death on failure with useful error message is probably in order.